gameplay_cpp/src/character/fsm/states/inair.hpp

```diff
@@ -24,10 +24,12 @@ class InAirState : public BaseState {
 	}
 
 	TState enter(Context& context) override {
+		context.anim->inAir();
 		return {};
 	}
 
 	TState exit(Context& context) override {
+		context.anim->onGround();
```

Comment:
Dette burde være i OnGround::enter istedenfor her. Vi vet teknisk sett ikke hvilken state som er neste state.

Reply:
Hvis dette er en overgang som krever å vite at nåværende state er InAir, for å gjøre InAir->OnGround animasjon overgang ordentlig, så tror jeg at context burde kanskje inneholde previous state. 

Har tenkt på å legge det til, men hadde ikke et spesifikt tilfelle i tankene enda. Animasjons overganger er kanskje det.

Reply:
Selvom jeg ser at her så bruker du dette for å resette blend. Så kanskje ikke super relevant for dette spesifike tilfelle. 
Forrige kommentar blir mer konseptuelt hvordan jeg tenker vi burde gjøre animasjoner i state maskinen.

Reply:
Så ikke at det var en onground state lol, men det er fikset nå!

Reply:
Men jeg tror kontekst burde inneholde forrige state ja. 
For eksempel ved en parry, så kan vi sette animasjon til parry, men dersom vi entret fra luften vil vi helst at beina fortsatt skal være "in air".
Dersom vi gjorde det fra ground vil vi helst at beina skal gå i en parry stance fremfor stille, så her krever vi også info om forrige state

Reply:
Liker ideen om en state stack! Kan ikke se for meg mer enn 2 states ennå, tho

Reply:
Gjort! Denne branchen er rebaset på det som legger til stacken